src/Components/Nav/Navdata/Eventdata.js

```diff
@@ -0,0 +1,5 @@
+export const Eventdata = [
+  {
+    content: ["이벤트", "기획전", "아울렛"],
+  },
+];
```

Comment:
```suggestion
export const Eventdata = ["이벤트", "기획전", "아울렛"];
```
간단하게 데이터 구조를 만들어줄 수 있지 않을까요?